styles.css

```diff
@@ -0,0 +1,51 @@
+body{
+    background: url("./images/chris-anderson-8VPSh4Wj61Q-unsplash.jpg");
+    /* background: url("https://images.unsplash.com/photo-1566305977571-5666677c6e98?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=345&q=80"); */
```

Comment:
- Please make sure to remove comments from your codebase as it will make your code look unclean.